driver/web/rest/main.go

```diff
@@ -202,6 +202,8 @@ type joinGroupExtApprovement struct {
 	GroupName                string    `json:"group_name"`
 	FirstName                string    `json:"first_name"`
 	LastName                 string    `json:"last_name"`
+	Email                    string    `json:"email"`
```

Comment:
It is better to use a pointer *string for phone and email because when the field data is not presented then it should be null instead of "" (empty string).